procedures/release-duty/index.rst

```diff
@@ -36,6 +36,17 @@ tools
 Communication
 -------------
 
+Slack Channels
```

Comment:
Suggested by @tomprince in `Process and Handling of Firefox-CI administration requests` document.